lib/search.js

```diff
@@ -1,81 +1,63 @@
 'use strict';
 
 const request = require('./utils/request');
-const cheerio = require('cheerio');
-
-function getClp (html) {
-  // Try to find clp from "next page" html elem.
-  let match = html.match(/\?clp=(.*?)">/);
-  // ... if we don't have it, we're probably on innerPage;
-  // try to parse it from search_collection_more_results_cluster instead
-  // var curl='https://play.google.com/store/apps/collection/search_collection_more_results_cluster?clp\x3dggENCgVwYW5kYRABGgIIAA%3D%3D:S:ANO1ljKV8KM';
-  if (!match) match = html.match(/\?clp\\x3d(.*?)';/);
-  return match && match[1].replace(/%3D/g, '=');
-}
+const scriptData = require('./utils/scriptData');
+const mapping = require('./utils/appListMapping');
+const R = require('ramda');
 
-function getNextToken (html) {
-  // extract the token for the next page request
-  // for the record, I hate regexps
-  // const s = html.match(/\\42(GAE.+?)\\42/);
-  const s = html.match(/\\x22-p6(.*?):S:(.*?)\\x22/g);
-  if (!s) {
-    return undefined;
-  }
-  return s[0].replace(/\\\\u003d/g, '=').replace(/\\x22/g, '');
-}
+const body = 'f.req=%5B%5B%5B%22qnKhOb%22%2C%22%5B%5Bnull%2C%5B%5B10%2C%5B10%2C50%5D%5D%2Ctrue%2Cnull%2C%5B96%2C27%2C4%2C8%2C57%2C30%2C110%2C79%2C11%2C16%2C49%2C1%2C3%2C9%2C12%2C104%2C55%2C56%2C51%2C10%2C34%2C77%5D%5D%2Cnull%2C%5C%22%token%%5C%22%5D%5D%22%2Cnull%2C%22generic%22%5D%5D%5D';
```

Comment:
what is this exactly? can't it be extracted from the html instead of hardcoding it here?

Reply:
I don't know where it can be extracted. If you find I can fix it.